default.json

```diff
@@ -21,24 +23,37 @@
   "postUpdateOptions": [
     "pnpmDedupe"
   ],
-  "separateMinorPatch": true,
```

Comment:
We have multiple configuration that seem to be contradictory, I think its probably better to merge patch versions into single PRs.